.github/workflows/source_aws.yml

```diff
@@ -50,6 +50,8 @@ jobs:
         uses: cloudquery/setup-cloudquery@v3
         with:
           version: v3.23.1
+      - name: vendor
```

Comment:
What's the reason we need this?

Reply:
`gen` step (`make gen` -> `make gen-spec-schema`) requires code to be present to load comments from AWS SDK

Reply:
I can move `go mod vendor` to `gen-spec-schema` step, it will have the same effect

Reply:
Moved in 11e4350